test/controllers/VatDetailsControllerSpec.scala

```diff
@@ -180,12 +188,50 @@ class VatDetailsControllerSpec extends ControllerBaseSpec {
           (mockAuditService.audit(_: AuditModel, _: String)(_: HeaderCarrier, _: ExecutionContext))
             .stubs(*, *, *, *)
             .returns({})
+
+          (mockMandationService.getMandationStatus(_: String)(_: HeaderCarrier, _: ExecutionContext))
+            .expects(*, *, *)
+            .returning(mandationStatusServiceResult).once()
         }
 
         val result: Future[Result] = target.details()(fakeRequest)
         status(result) shouldBe Status.OK
       }
     }
+
+    "the feature switch is turned off" should {
+      "return a VatDetailsViewModel as a MTDfB user" in new DetailsTest {
```

Comment:
Saying 'return a VatDetailsViewModel' etc doesn't really reflect what the tests are asserting